api/src/services/assessmentsDummyService.ts

```diff
@@ -0,0 +1,185 @@
+import db from './db';
+
+interface ProgramParticipantsRow {
+  id: number;
+  principal_id?: number;
+  program_id?: number;
+  role_id?: number;
+}
+
+/**
+ * Return  program_participants table for any rows matching this principalId and this programId.
```

Comment:
Not quite. We're returning matching rows from the `program_participants` table that have either been inserted or updated based on their pre-existence in the table.